### PR TITLE
Validation renderer

### PR DESCRIPTION
**Pull request by Ullfis**
removing unused label classes + lint fix
export MaterializeFormValidationRenderer from bridge library

### CODE REVIEW

**Comment by eikaramba** [comment]
i think length is not there anymore. use minLength instead.

**Comment by eikaramba** [comment]
and another change:

validate() now returns a promise. So use that instead

```
this.controller.validate().then(v => {
      if (v.length === 0) {
        //success -> register the user or whatever
      }else{
        this.myErrors = v;
      }
    });
```

**Comment by Ullfis** [comment]
Thank you :smile:

File: src/validation/validationRenderer.js [hunk 1]
```diff
@@ -1,80 +1,63 @@
-import { inject } from 'aurelia-dependency-injection';
-import { validationRenderer } from 'aurelia-validation';
-
-@validationRenderer
```
Comment (Thanood) [inline src/validation/validationRenderer.js]: This is actually not needed? Wow, I need to learn a few things.. :smile:

**Pull request merged**